plonetheme/barceloneta/theme/less/variables.plone.less

```diff
@@ -181,7 +181,7 @@
 //** Border color for inputs on focus
 @plone-input-border-focus:             @plone-portlet-list-bullet;
 //** Placeholder text color
-@plone-input-color-placeholder:        @plone-gray-light;
+@plone-input-color-placeholder:        @plone-gray-lighter;
```

Comment:
I would revert this back to `@plone-gray-light`
and use `@plone-input-color:                    @plone-gray-dark;` above.